readthedocs/oauth/admin.py

```diff
@@ -86,8 +86,25 @@ class RemoteRepositoryRelationAdmin(admin.ModelAdmin):
     list_select_related = (
         "remote_repository",
         "user",
+        "account",
+    )
+    list_display = (
+        "id",
+        "remote_repository",
+        "user",
+        "account",
+        "vcs_provider",
```

Comment:
Isn't it possible to use the following and avoid having to define the method below?

```suggestion
        "remote_repository__vcs_provider",
```

Reply:
nope :/ django doesn't like it, or at least in 4.2, I see that 5.2 has that case as a valid example https://docs.djangoproject.com/en/5.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display